spec/features/invite_spec.rb

```diff
@@ -12,23 +12,30 @@
     sign_in tom
 
     visit invite_path
-    fill_in :invitation_email, with: jerrys_email
-    fill_in :invitation_full_name, with: jerrys_name
+    fill_in "Friend's name", with: jerrys_name
+    fill_in "Friend's Email Address", with: jerrys_email
+    fill_in 'Your message', with: 'Hello.'
+
     click_button 'Send Invitation'
-    open_email jerrys_email
+    expect(page).to have_content('Your friend has been invited')
+    # sign out the current user Tom ...
+    sign_out tom
```

Comment:
If you need to sign out after each spec, something is wrong. Your other specs should be isolated and not be impacted by this. Otherwise it feels like a spam to me.